nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -972,11 +973,15 @@ public Integer getFlowElectionMaxCandidates() {
     }
 
     public String getFlowConfigurationArchiveMaxTime() {
-        return getProperty(FLOW_CONFIGURATION_ARCHIVE_MAX_TIME, DEFAULT_FLOW_CONFIGURATION_ARCHIVE_MAX_TIME);
+        return getProperty(FLOW_CONFIGURATION_ARCHIVE_MAX_TIME, null);
```

Comment:
@ijokarumawak i think for each of these max values we should have some sensible default.  For time perhaps 14 days.  For storage perhaps 1GB.  For count 50.